cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('Login and access profile', () => {
+  it('login', () => {
+    cy.visit('https://emel-h.github.io/social-media-client/')
```

Comment:
```suggestion
    cy.visit('/')
```

Use the cypress server